files/ru/web/html/element/heading_elements/index.md

```diff
@@ -25,7 +25,7 @@ translation_of: Web/HTML/Element/Heading_Elements
 - Информация о заголовке может использоваться пользовательскими агентами, например, для автоматического создания оглавления для документа
 - Не используйте низкие уровни чтобы снизить размер шрифта: используйте [CSS](/ru/docs/Web/CSS) {{cssxref("font-size")}} вместо.
 - Избегайте пропуск уровней заголовков: всегда начинайте с `<h1>`, потом используйте `<h2>`, и так далее.
-- Вам следует рассмотреть избегание использования \<h1> более раза на страницу. Смотрите {{SectionOnPage("/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines", "Defining sections")}}.
```

Comment:
The last sentence is not existed in en-US.

![image](https://github.com/mdn/translated-content/assets/15844309/c4155cad-976f-47ee-8d70-efad72c99b4f)

Reply:
It is actually the information that is described in the next section in the original article. I has opened the issue on syncronization Russian translation with original article https://github.com/mdn/translated-content/issues/13480